build.gradle

```diff
@@ -12,7 +12,7 @@ version '0.12.1'
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
     version 'IU-LATEST-EAP-SNAPSHOT'
-    plugins = ['JavaScriptLanguage', 'CSS', 'PsiViewer:201.3803.71-EAP-SNAPSHOT.1']
+    plugins = ['JavaScriptLanguage', 'CSS', 'platform-images', 'PsiViewer:201.3803.71-EAP-SNAPSHOT.1']
```

Comment:
still, without the change tests don't work for me

Reply:
I've bumped stuff according to https://github.com/JetBrains/intellij-platform-plugin-template, I guess `gradle-intellij-plugin` fixes dependency resolving, so this change is no longer necessary

Reply:
👍